test/test_p4est_2d.jl

```diff
@@ -390,6 +390,14 @@ end
         # Corresponding issue: https://github.com/trixi-framework/Trixi.jl/issues/1877
         @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 15000
     end
+
+    # Test `resize!`
+    ode_alg = Trixi.SimpleSSPRK33(stage_callbacks = stage_callbacks)
+    integrator = Trixi.init(ode, ode_alg, dt = 42.0, callback = callbacks)
+    resize!(integrator, 42)
+    @test length(integrator.u) == 42
+    @test length(integrator.du) == 42
+    @test length(integrator.u_tmp) == 42
```

Comment:
```suggestion
    @test length(integrator.u_tmp) == 42
    semi = integrator.p
    @test length(semi.cache.antidiffusive_fluxes) == 42
    (; limiter) = semi.volume_integral
    @test length(limiter.cache.subcell_limiter_coefficients) == 42
```
And maybe we should test this resize routine also without SubcellLimiting such that an empty version of `resize!(semi, volume_integral, new_size)` is tested.

Reply:
Don't you think we should also test the correct size of `antidiffusive_fluxes` and `subcell_limiter_coefficients` for the subcell limiting case, like I did above?